packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -705,29 +705,25 @@ export const RemoteServerFactory = {
                 },
                 onProxyRes: (proxyRes, req) => {
                     if (proxyRes.statusCode && proxyRes.statusCode >= 400) {
-                        logger.error(
-                            `Failed to proxy SLAS Private Client request - ${proxyRes.statusCode}`,
-                            {
-                                namespace: '_setupSlasPrivateClientProxy',
-                                additionalProperties: {statusCode: proxyRes.statusCode}
-                            }
-                        )
-                        logger.error(
-                            `Please make sure you have enabled the SLAS Private Client Proxy in your ssr.js and set the correct environment variable PWA_KIT_SLAS_CLIENT_SECRET.`,
-                            {namespace: '_setupSlasPrivateClientProxy'}
-                        )
-                        logger.error(
-                            `SLAS Private Client Proxy Request URL - ${req.protocol}://${req.get(
-                                'host'
-                            )}${req.originalUrl}`,
-                            {
-                                namespace: '_setupSlasPrivateClientProxy',
-                                additionalProperties: {
-                                    protocol: req.protocol,
-                                    originalUrl: req.originalUrl
+                        proxyRes.on('data', (data) => {
```

Comment:
Do we have an concern here about the following:

1. Are we sure that the data even is always called? e.g. can their be a failure that doesn't have a body on it?
2. Is there the potential to have errors if there is similar code downstream that is trying to access the data event in the same way, possibly resulting in the downstream callback not firing?
3. In question 1 above we wouldn't make any logging calls if there is a scenario where the data event is not fired. Do you think it makes sense that we have a logger call outside of this data callback? 
4. Do we know that the data is going to be the full message, e.g. does the data event only ever get called once? See [this](https://nodejs.org/en/learn/modules/anatomy-of-an-http-transaction#request-body) example of using the data event.. maybe we need to do something like that.

Reply:
It also looks like we are losing all the setup errors. Do we not find these valuable? Could we use those as a way to help customers that are misconfigured.

Reply:
Good questions @bendvc 

1. All responses from SLAS should have a body, even the HTTP 500s. Example 400 and 500 responses are shown in [the SLAS docs](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=getAccessToken)
2. Admittedly, I am not the most familiar with the data event - I just wanted a way to log the error message returned by the SLAS response in the body. So I cannot say if it will have any downstream effects or not.
3. I've thought about having a logger outside of the data event but opted to consolidate everything into a single log entry for easier searching in Log Center
4. I can give that a try.
5. I feel that the setup errors here are redundant since we already return setup related errors elsewhere (see `_handleMissingSlasPrivateEnvVar`) and currently this message is misleading since we are outputting it on every instance where SLAS returns a 400 or 500 and swallowing the original message.